docs/development/design-decisions.md

```diff
@@ -2,6 +2,17 @@
 
 This file documents the design choices as well as configuration options.
 
+## Intended usage by end-users
+
+Renovate is intended to be used as an app.
+It should not be used as a library, because it lacks a stable API.
+
+We do not want to create a stable API, or document our existing API.
```

Comment:
We want a stable api some time, but currently it's changing too often to be stable soon.

Reply:
Do we want to make any promises/future work by suggesting we're going to stabilize our API? 

@rarkins What do you think?

Reply:
```suggestion
Right now we do not have a stable API.
```